### PR TITLE
noelle02 update

### PR DESCRIPTION
**Pull request by Noelle-KH**
noelle02 update

### CODE REVIEW

File: controllers/todoController.js [hunk 1]
```diff
@@ -66,6 +66,9 @@ let todoController = {
           .catch(error => console.log(error))
       })
       .catch(error => console.log(error))
+  },
+  getTodo: (req, res, next) => {
```
Comment (Lilynews) [inline controllers/todoController.js:70]: yoooooo

Comment (Noelle-KH) [reply controllers/todoController.js:70]: yooooooo

Comment (Lilynews) [reply controllers/todoController.js:70]: new topic

**Review by Lilynews — COMMENTED** [review]
new topic-1

**Review by Lilynews — COMMENTED** [review]
new topic-2

File: controllers/todoController.js [hunk 2]
```diff
@@ -66,6 +66,9 @@ let todoController = {
...
+    console.log('noelle02 update')
   }
 }
 module.exports = todoController
```
[the remark this answers is not in the document]
Comment (Noelle-KH) [reply controllers/todoController.js:74]: 我改好了

[the remark this answers is not in the document]
Comment (Lilynews) [reply controllers/todoController.js:74]: okkk

**Review by Lilynews — COMMENTED** [review]
check

File: controllers/todoController.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (Lilynews) [reply controllers/todoController.js:74]: approve

**Review by Lilynews — APPROVED** [review]
approve-1

**Pull request merged**